spec/param_parser_spec.cr

```diff
@@ -2,101 +2,98 @@ require "./spec_helper"
 
 describe "ParamParser" do
   it "parses query params" do
-    route = Route.new "POST", "/" do |env|
-      hasan = env.params.query["hasan"]
-      "Hello #{hasan}"
-    end
     request = HTTP::Request.new("POST", "/?hasan=cemal")
     query_params = Kemal::ParamParser.new(request).query
     query_params["hasan"].should eq "cemal"
   end
 
-  it "parses multiple values for query params" do
-    route = Route.new "POST", "/" do |env|
-      hasan = env.params.query["hasan"]
-      "Hello #{hasan}"
+  context "#query" do
+    it "parses multiple values for query params" do
+      request = HTTP::Request.new("POST", "/?hasan=cemal&hasan=lamec")
```

Comment:
This is valid and really dependent on language / platform e.g 

Here's a good answer on this topic http://stackoverflow.com/a/9547490/911870